sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -83,16 +83,19 @@ abstract class StringRegexExpression extends BinaryExpression
           % matches zero or more characters in the input (similar to .* in posix regular
           expressions)
 
-          The escape character is '\'. If an escape character precedes a special symbol or another
-          escape character, the following character is matched literally. It is invalid to escape
-          any other character.
+          The default escape character is '\'. If an escape character precedes a special symbol
+          or another escape character, the following character is matched literally. It is
+          invalid to escape any other character.
```

Comment:
How about just moving this section into the description for `escape`?

Reply:
Also update the `usage` section.

Reply:
OK. I changed it.